tests/data/sourcegit/source_git/beer.spec

```diff
@@ -0,0 +1,18 @@
+Name:           beer
```

Comment:
Can we put the spec file in a directory? To mimic the most likely behaviour when people store the complete dist-git content in a dedicated dir.